test/dotnet-new.Tests/DotnetNewDetailsTest.Approval.cs

```diff
@@ -177,5 +185,32 @@ public Task CanDisplayDetails_InstalledPackage_FolderInstallation()
             return Verify(commandResult.StdOut)
                 .AddScrubber(output => output.ScrubAndReplace(basicFSharp, "%TEMPLATE FOLDER%"));
         }
+
+        private async Task<string> GetLatestVersion(string packageName)
+        {
+            using (HttpClient client = new HttpClient())
+            {
+                string json = await client.GetStringAsync($"https://api.nuget.org/v3-flatcontainer/{packageName}/index.json");
+                JObject obj = JObject.Parse(json);
+
+                var versions = obj["versions"]?.ToObject<List<string>>();
+                if (versions == null || versions.Count == 0)
+                {
+                    throw new Exception("No versions found.");
+                }
+
+                return versions.Last();
```

Comment:
FYI the last entry is not necessarily the latest stable version, e.g. when we have previews, but looks like we don't need to care about that since templates don't support preview versions: https://github.com/dotnet/templating/issues/1423